mautic/mautic
# M3: Refactor Deprecations - QueueBundle

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8003
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR refactors QueueBundle to be compatible with Symfony 3.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Run the tests like `php bin/phpunit app/bundles/QueueBundle/Tests`
3. Run `php app/console mautic:queue:process`
4. Yous should observe no errors in both cases.
